sdv/multi_table/hma.py

```diff
@@ -266,6 +348,7 @@ def _extract_parameters(self, parent_row, table_name, foreign_key):
         return flat_parameters.rename(new_keys).to_dict()
 
     def _recreate_child_synthesizer(self, child_name, parent_name, parent_row):
+        # When more than one foreign key exists between two tables, only the first one is modeled.
```

Comment:
I don't think this is right. I think the root is extended with ALL of the parameters for children, but only one of them is used to create the child table synthesizer. See https://github.com/sdv-dev/SDV/blob/8b95b135c65a4e29b3741bcf660dd8d4264eb405/sdv/multi_table/hma.py#L152-L169